examples/asset/texture_sampler.rs

```diff
@@ -0,0 +1,120 @@
+        ..default()
+    });
+
+    // Texture from ambientCG.com, licensed under the Creative Commons CC0 1.0 Universal License.
+    // https://ambientCG.com/a/Facade018A
+
+    // By default Bevy loads images to textures with sampler settings that clamp the image to the edges.
+    let image = asset_server.load("textures/facade018a.png");
+
+    // Here we override the sampler settings to repeat the image instead.
+    let image_repeat = asset_server.load_with_settings(
+        // We are using another file name, because Bevy ignores different loader settings for the same file.
+        // https://github.com/bevyengine/bevy/issues/11111
+        "textures/facade018a_copy.png",
+        |s: &mut ImageLoaderSettings| match &mut s.sampler {
+            ImageSampler::Default => {
+                s.sampler = ImageSampler::Descriptor(ImageSamplerDescriptor {
+                    address_mode_u: ImageAddressMode::Repeat,
+                    address_mode_v: ImageAddressMode::Repeat,
+                    ..default()
+                });
+            }
+            ImageSampler::Descriptor(sampler) => {
+                sampler.address_mode_u = ImageAddressMode::Repeat;
+                sampler.address_mode_v = ImageAddressMode::Repeat;
+            }
+        },
+    );
+
+    // Instead of using a standard quad with UV coordinates in the range `0..=1` (spanning the entire texture),
+    // we create a quad with UV coordinates in the range `-1..=2`, going beyond the texture by one entire unit in each direction.
+    // Texture behaviour then depends on the selected sampler mode.
+    // By default (`ImageAddressMode::ClampToEdge`) the out-of-bounds UV coordinates will map to the edge of the texture.
+    // When `ImageAddressMode::Repeat` is specified the out-of-bounds UV coordinates will repeat the texture.
+    let mesh = meshes.add(quad_with_custom_uv(-1.0, 2.0));
+
+    commands.spawn(MaterialMesh2dBundle {
+        mesh: mesh.clone().into(),
+        material: materials.add(ColorMaterial {
+            texture: Some(image),
+            ..default()
+        }),
+        transform: Transform::from_translation(Vec3::new(-0.95, -0.45, 0.))
+            .with_scale(Vec3::new(0.9, 0.9, 0.9)),
+        ..default()
+    });
+
+    commands.spawn(MaterialMesh2dBundle {
+        mesh: mesh.into(),
+        material: materials.add(ColorMaterial {
+            texture: Some(image_repeat),
+            ..default()
+        }),
+        transform: Transform::from_translation(Vec3::new(0.05, -0.45, 0.))
+            .with_scale(Vec3::new(0.9, 0.9, 0.9)),
+        ..default()
+    });
+}
+
+/// Creates a quad with UV coordinates in range `uv_low..=uv_high`, instead of the usual UV range `0..=1`.
+fn quad_with_custom_uv(uv_low: f32, uv_high: f32) -> Mesh {
+    let mut mesh = Mesh::new(PrimitiveTopology::TriangleList);
+    mesh.insert_attribute(
+        Mesh::ATTRIBUTE_POSITION,
+        vec![[0., 0., 0.], [1., 0., 0.], [1., 1., 0.], [0., 1., 0.]],
```

Comment:
Modifying the default Quad might be better, I'll at least fix the mesh coordinates.

A better function name is welcome, although `repeating_quad(times: f32)` feels a little suggestive, because it suggest using this mesh would automatically cause repeating, whereas it only sets up a mesh that can be repeated over, but maybe this is not such a problem given the comments explaining the mechanisms.